frontend/src/modules/Omics/views/OmicsRunCreateForm.js

```diff
@@ -0,0 +1,503 @@
+                  color="textPrimary"
+                  component={RouterLink}
+                  to="/console/omics/runs/new"
+                  variant="subtitle2"
+                >
+                  Create Run
+                </Link>
+              </Breadcrumbs>
+            </Grid>
+            <Grid item>
+              <Box sx={{ m: -1 }}>
+                <Button
+                  color="primary"
+                  component={RouterLink}
+                  startIcon={<ArrowLeftIcon fontSize="small" />}
+                  sx={{ mt: 1 }}
+                  to="/console/omics"
+                  variant="outlined"
+                >
+                  Cancel
+                </Button>
+              </Box>
+            </Grid>
+          </Grid>
+          <Box sx={{ mt: 3 }}>
+            <Formik
+              initialValues={{
+                workflowUri: omicsWorkflow.workflowUri,
+                label: '',
+                SamlAdminGroupName: '',
+                environment: '',
+                destination: '',
+                parameterTemplate: omicsWorkflow.parameterTemplate
+              }}
+              validationSchema={Yup.object().shape({
+                workflowUri: Yup.string()
+                  .max(255)
+                  .required('*Workflow is required'),
+                label: Yup.string().max(255).required('*Run Name is required'),
+                parameterTemplate: Yup.string()
+                  .max(5000)
+                  .required('*Parameters are required'),
+                SamlAdminGroupName: Yup.string()
+                  .max(255)
+                  .required('*Team is required'),
+                environment: Yup.object().required('*Environment is required'),
+                destination: Yup.string()
+                  .max(255)
+                  .required('*Destination is required')
+              })}
+              onSubmit={async (
+                values,
+                { setErrors, setStatus, setSubmitting }
+              ) => {
+                await submit(values, setStatus, setSubmitting, setErrors);
+              }}
+            >
+              {({
+                errors,
+                handleBlur,
+                handleChange,
+                handleSubmit,
+                isSubmitting,
+                setFieldValue,
+                touched,
+                values
+              }) => (
+                <form onSubmit={handleSubmit} {...props}>
+                  <Grid container spacing={3}>
+                    <Grid item lg={7} md={6} xs={12}>
+                      <Card sx={{ mb: 3 }}>
+                        <CardHeader title="Details" />
+                        <CardContent>
+                          <TextField
+                            disabled
+                            fullWidth
+                            label="Workflow Id"
+                            name="workflowUri"
+                            value={omicsWorkflow ? omicsWorkflow.id : ''}
```

Comment:
should this be `omicsWorkflow.id` or `omicsWorkflow.workflowUri`? I see we say its `workflowUri` and use it as such in `def create_omics_run()` in backend but we set it as `.id` here

Reply:
We have renamed this to avoid confusion; we are using the workflowUri for everything; workflowId is just for the AWS API call